modules/raycast/SCsub

```diff
@@ -67,7 +67,7 @@ if env["builtin_embree"]:
     env_raycast.AppendUnique(CPPDEFINES=["NDEBUG"])  # No assert() even in debug builds.
 
     if not env.msvc:
-        if env["arch"] == "x86_64":
+        if env["arch"] in ["x86_64", "x86_32"]:
```

Comment:
Note for myself, this is a problem for Linux distros with a `ix86` arch that still supports machines without SSE2 support (e.g. Debian and Mageia). Such machines probably wouldn't have a good experience running Godot anyway, but we might still want to make this opt-out or builds would outright crash there.